frontend/src/main/web/src/components/Tooltip/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import { isRequiredForA11y } from 'react-prop-types'
+import { capitalize } from 'lodash'
+import { colors } from '../../constants/styles'
+
+const Tooltip = ({
```

Comment:
Needs doc. The name is enough to tell me that this will show a popup text box positioned near the element, but the examples in `readme.md` make it look like you should always use them with an `OverlayTrigger` to get the behaviour for when it shows and hides. Doc should mention that so that people don't waste time trying to make it work without that.